From 46bab70ba402dd6faeae0a9d7aaa7372f1f1f2ab Mon Sep 17 00:00:00 2001
From: Changhyeok Bae <changhyeok.bae@gmail.com>
Date: Thu, 6 Jul 2017 09:41:31 +0900
Subject: [PATCH] Disable eglfs-brcm explicitly

:Release Notes:
Disable eglfs-brcm explicitly

:Detailed Notes:
After applying userland for omxplayer, eglfs-brcm is built.
However we don't need eglfs-brcm because webOS uses drm and eglfs for
qpa.

:Testing Performed:
Only build tested.

:QA Notes:

:Issues Addressed:
[PLAT-42918] qtbase build failures on apollo-rpi3 no shared state build

Change-Id: I9392e5aec929fe7d1c3dc9f52f94323ede3f86e0
---
 src/gui/configure.json                                     | 7 +------
 .../eglfs/deviceintegration/deviceintegration.pro          | 1 -
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/src/gui/configure.json b/src/gui/configure.json
index 864603429a..a15e9c4c0a 100644
--- a/src/gui/configure.json
+++ b/src/gui/configure.json
@@ -1118,11 +1118,6 @@
             "condition": "!config.android && !config.darwin && !config.win32 && features.egl",
             "output": [ "privateFeature" ]
         },
-        "eglfs_brcm": {
-            "label": "EGLFS Raspberry Pi",
-            "condition": "features.eglfs && tests.egl-brcm",
-            "output": [ "privateFeature" ]
-        },
         "eglfs_egldevice": {
             "label": "EGLFS EGLDevice",
             "condition": "features.eglfs && tests.egl-egldevice && features.kms",
@@ -1636,7 +1631,7 @@ QMAKE_LIBDIR_OPENGL[_ES2] and QMAKE_LIBS_OPENGL[_ES2] in the mkspec for your pla
                     "section": "EGLFS details",
                     "condition": "features.eglfs",
                     "entries": [
-                        "eglfs_openwfd", "eglfs_viv", "eglfs_viv_wl", "eglfs_rcar", "eglfs_egldevice", "eglfs_gbm", "eglfs_mali", "eglfs_brcm", "egl_x11"
+                        "eglfs_openwfd", "eglfs_viv", "eglfs_viv_wl", "eglfs_rcar", "eglfs_egldevice", "eglfs_gbm", "eglfs_mali", "egl_x11"
                     ]
                 },
                 "linuxfb", "vnc", "mirclient",
diff --git a/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro b/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
index 92ee83fd8f..7ca4b5378d 100644
--- a/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
+++ b/src/plugins/platforms/eglfs/deviceintegration/deviceintegration.pro
@@ -4,7 +4,6 @@ QT_FOR_CONFIG += gui-private
 qtConfig(egl_x11): SUBDIRS += eglfs_x11
 qtConfig(eglfs_gbm): SUBDIRS *= eglfs_kms_support eglfs_kms
 qtConfig(eglfs_egldevice): SUBDIRS *= eglfs_kms_support eglfs_kms_egldevice
-qtConfig(eglfs_brcm): SUBDIRS += eglfs_brcm
 qtConfig(eglfs_mali): SUBDIRS += eglfs_mali
 qtConfig(eglfs_viv): SUBDIRS += eglfs_viv
 qtConfig(eglfs_rcar): SUBDIRS += eglfs_rcar
